### PR TITLE
Add  information on a report's Image Resources

### PR DESCRIPTION
**Pull request by MarySammal**

### CODE REVIEW

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md [hunk 1]
```diff
@@ -15,25 +15,52 @@ To add the **Picture Box** control to a report, drag the **Picture Box** item fr
 
 Specify one of the following properties to set an image:
 
-- **ImageSource**  
-    Use this property to save images along with a report definition.
-- **ImageUrl**  
-    Use this property to save only the path to the image.
+- **Image Source**  
+    Save the image to the report's definition.
```
Comment (albertov05) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md]: Change "report's definition" to "report definition"

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md [hunk 2]
```diff
@@ -15,25 +15,52 @@ To add the **Picture Box** control to a report, drag the **Picture Box** item fr
...
+- **Image Url**  
+    Save only a path to the image.
```
Comment (albertov05) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md]: Change "Save only" to "Only save"

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md [hunk 3]
```diff
@@ -15,25 +15,52 @@ To add the **Picture Box** control to a report, drag the **Picture Box** item fr
...
 
 ![](../../../../../images/eurd-win-picture-box-image-property.png)
 
+## Assign an External Image
+
+Click the **Image Source** / **Image URL** property's ellipsis button and the **Open File** dialog is invoked.
```
Comment (albertov05) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md]: Change "and the **Open File** dialog is invoked" to "to invoke the **Open File** dialog"

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md [hunk 4]
```diff
@@ -15,25 +15,52 @@ To add the **Picture Box** control to a report, drag the **Picture Box** item fr
...
+
+![](../../../../../images/eurd-win-ImageSource-OpenFileDialog.png)
+
+The selected image / its URL is saved to the report definition **.repx** file.
```
Comment (albertov05) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md]: Do you mean "The selected image and its URL are saved..."

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md [hunk 5]
```diff
@@ -15,25 +15,52 @@ To add the **Picture Box** control to a report, drag the **Picture Box** item fr
...
+
+## Assign an Image from the Report's Image Collection
+
+Set the report's **Image Resources** property.
+
+![](../../../../../images/eurd-win-ImageResources-Editor.png)
+
+Invoke the **Picture Box**'s smart tag. Click the **Expression** option's ellipsis button to invoke the **Expression Editor**. Choose an image from the **Images** collection:
```
Comment (albertov05) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md]: "Invoke the **Picture Box**'s smart tag."
I don't think you can invoke a smart tag. Are you clicking the smart tag to invoke a context menu, possibly?

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md [hunk 6]
```diff
@@ -15,25 +15,52 @@ To add the **Picture Box** control to a report, drag the **Picture Box** item fr
...
+
+![ImageSource-OpenFileDialog](../../../../../images/eurd-win-ImageSource-ExpressionEditor-ImagesCollection.png)
+
+
 ## Bind a Picture Box to Data
-You can use the Picture Box to display an image [dynamically obtained](../../bind-to-data/bind-controls-to-data-expression-bindings.md) from a data source. Click the control's smart tag, expand the **Image** property's **Expression** drop-down list and select the data field.
 
-![](../../../../../images/eurd-win-picture-box-bind-to-data.png)
+Use one of the following techniques to add the **Picture Box** control that obtains an image from a data source.
```
Comment (albertov05) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md]: Change "the **Picture Box** control that obtains an image from a data source" to "the **Picture Box** control, which obtains an image from a data source"

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md [hunk 7]
```diff
@@ -15,25 +15,52 @@ To add the **Picture Box** control to a report, drag the **Picture Box** item fr
...
+
+- Invoke the control's smart tag, expand the **Expression** drop-down list for the **Image Source** property and select a data field.
```
Comment (albertov05) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md]: Invoke the control's smart tag
Are you actually clicking the smart tag to invoke its context menu? Please revise as necessary.

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md (hunk 4 above)
Comment (MarySammal) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md]: No, / means or

"Click the **Image Source** / **Image URL** property's ellipsis button...

The selected image / its URL is saved..."

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md (hunk 5 above)
Comment (MarySammal) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md]: I agree, thanks!

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md (hunk 4 above)
Comment (albertov05) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md]: Then we can say "The selected image or its URL is saved..."

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md (hunk 5 above)
Comment (albertov05) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md]: So can we say "Click the **Picture Box** smart tag to invoke the context menu."

**Pull request merged**